src/Uno.UI/UI/Xaml/WindowManagerInterop.wasm.cs

```diff
@@ -1522,9 +1542,25 @@ internal enum HtmlPointerButtonUpdate
 #if NET7_0_OR_GREATER
```

Comment:
I think it should be possible to get the benefits when not targeting .NET 7.

Two options I can think of:

#### Get the benefits when using .NET 7 SDK, regardless of the target framework.

The `JSImportGenerator` ships with the SDK, and I think it's quite unrelated to the TFM.
So we can define `JSImportAttribute` when `!NET7_0_OR_GREATER`

In this case, we'll need to only use the optimized code path when the .NET SDK is 7.0+.

#### Always get the benefits

This can be achieved by porting the `JSImportGenerator` implementation here.

Reply:
We discussed about this internally, and while it's possible, it's an additional level of complexity (and debugging) that will not be serving many developers (a Wasm head targeting `net6.0` with boostrapper 7.0.x where upgrading to `net7.0` is very easy).